cdap-app-fabric/src/main/java/co/cask/cdap/app/guice/DistributedProgramRunnableModule.java

```diff
@@ -113,6 +115,7 @@ protected void configure() {
             .in(Scopes.SINGLETON);
           // bind AuthorizationEnforcer to AuthorizationEnforcementService
           bind(AuthorizationEnforcer.class).to(AuthorizationEnforcementService.class).in(Scopes.SINGLETON);
+          bind(PrivilegesFetcher.class).to(RemotePrivilegesFetcher.class);
```

Comment:
right now, yes. But it will in a future PR start being used across the board. These mappings will then likely move to their own guice module, but that's for future.